### DISCUSSION TITLE
Home Run ⚾️

### DISCUSSION THREAD

**Discussion by 9mm** [nvim-mini/mini.nvim#548] [General]
I just wanted to say you have hit an absolute home run on `mini.files`

I have tried every oil-style plugin under the sun. The way you architected the very slim/lean windows that overlay at the top left is just *so freaking useable* and perfect.

The gap between you (#1) and 2nd place file explorer plugins is massive, worlds ahead.

I used to attempt to do all file editing in neotree but this just absolutely crushes

Thank you!!!

Im trying to think of what else would make it even better and TBH i feel like it does everything.

**Comment by echasnovski** [comment]
Thank you so much, Travis!

These kind words in the end of the long day are much appreciated :heart:
